testapi.pm

```diff
@@ -108,6 +108,31 @@ os-autoinst is used in the openQA project.
 
 =head1 internal
 
+=head2 calculate_clickpoint
+
+This is used to support several subroutines dealing with mouse clicks, do not call
+this from the tests.
+
+=cut    
+
+sub calculate_clickpoint {
+    # This is a non-exported helper function to calculate the clickpoints to limit repetition
+    # and redundancy of code.
```

Comment:
I'd merge this comment with the description of the function within the POD comment.